auth-web/src/routes/router.ts

```diff
@@ -247,13 +247,13 @@ export function getRoutes (): RouteConfig[] {
       component: EntityManagement,
       props: route => {
         try {
-          const base64Token = route.params.base64Token
-          const decodedToken = Base64.decode(base64Token) // Decode the Base64 token
-          const orgId = JSON.parse(decodedToken).fromOrgId // Extract the orgId from the decoded token
+          const base64Token = route.params.base64Token.split('.')[0]
```

Comment:
@Jxio FYI, I have deployed a fix for L245 which removes the double slash after acceptToken
`path: '/:base64OrgName/affiliationInvitation/acceptToken//:base64Token',`